### ISSUE TITLE
Remove the CLI config file

### ISSUE THREAD

**Issue by orta** [ds300/react-native-typescript-transformer#73]
👋 - I was looking at updating the docs for RN around the cli-config.js file, and in digging around in the RN source code, I noticed that we might not need to do that anymore.

Basically, we added the cli-config to ensure that [this code](https://github.com/facebook/react-native/blob/04aaa01991b2b6af9587f4edb6a897e05fc5e1f8/local-cli/dependencies/dependencies.js#L28-L33) could resolve to this dep. 

But passing in 
```
yarn start --transformer "node_modules/react-native-typescript-transformer/index.js" --sourceExts "ts"
``` 

should also work, and in an ideal case support in RN could be improved to handle  

```
yarn start --transformer "react-native-typescript-transformer" --sourceExts "ts"
```

Meaning a bit less process, it's a trade-off, and not one I'm sure I want to recommend yet - but worth noting

**Comment by VitorLuizC** [comment]
`--transformer "react-native-typescript-transformer"` looks for transformer in root instead of node_modules. Full transformer path is required otherwise it throws the error below.

```sh
ENOENT: no such file or directory, open '/MY_PATH/react-native-typescript-transformer'
```

The second problem is extensions also need `tsx`, without `tsx` it won't find any TypeScript component.

So, the right command would be

```sh
yarn start --transformer "node_modules/react-native-typescript-transformer/index.js" --sourceExts "ts,tsx"
```

**Comment by orta** [comment]
Yeah, I agree, I don't see much value also - I went and documented this file a bit on the site https://github.com/facebook/react-native-website/pull/462
